### ISSUE TITLE
Service down / long delivery delays?

### ISSUE THREAD

**Issue by fedus** [forwardemail/free-email-forwarding#177]
Hello,

I'd like to ask two questions:

1) Is it possible that the service is down at the time of writing? I have not received any mails in the last 1.5 days and friends have now told me about bounce backs. I can confirm that mails sent by myself to myself do not arrive anymore.

2) Quite often, it takes really long for mail to be delivered. It can range from a couple of minutes to more than a day - which is especially painful for confirmation emails, notifications etc. I can see in the headers that they were sent by the origin mail system at the time I would have expected them, but they get processed much later.

Happy to provide you with any information that could help you.

Thanks!

**Comment by niftylettuce** [comment]
Can you fwd me some emails, bounce backs, and your setup?  My email is
niftylettuce@gmail.com

service has been online
https://hetrixtools.com/r/4d2acef1653fa0bf0c32d87627939ac3/

**Comment by fedus** [comment]
Sorry for only answering now, I did not have the time to get back to you earlier. I appreciate your swift response.
I will email you with the headers of message when it happens next time.
Thanks for the great service.

**Comment by fedus** [comment]
I sent you an e-mail with headers from a 4h delivery delay example.

Thanks

**Comment by fedus** [comment]
Hey, just checking in to see whether you received my e-mail? Still experiencing the issues mentioned in the original post.

Thanks!

**Comment by fedus** [comment]
Just flagging that this issue is unresolved and should not be closed.

**Comment by niftylettuce** [comment]
How is it unresolved for you? Do you have a specific raw email not sending through? Email it to me? I would gladly provide support for you if you gave more insight.  Let me know, thanks!

**Comment by niftylettuce** [comment]
Just a heads up - the service pretty much has 100% uptime now, I'd say 99.99% uptime to be fair.  I have Cloudflare set up with monitoring and load balancers, I also get notifications of any packet loss or downtime, and any downtime on any server immediately falls over to another one until the server that has packet loss (or whatever the case) is back to being reported as healthy.

**Comment by niftylettuce** [comment]
There are two servers load balanced at the moment, mx1.forwardemail.net has a US SF server location and a UK London server location, and mx2.forwardemail.net is located in US NYC location.

**Comment by fedus** [comment]
Apologies for the delay of my answer - I only saw your replies today!

I sent you an example email where delivery took 4 days (see my comment of Nov 4 2019). Did you receive it?

In the meantime I also got a few people sending me bouncebacks from Gmail telling them that the message could not be delivered. Also, when I lock in to certain websites (Kickstarter for instance) I see a warning to change my email because messages bounced back).

I am happy to provide you more information in addition to that of Nov 4 of course.

I think it must be something else than a downtime problem anyway, indeed I don't think the service would be down for 4 days. 

Thanks again for your efforts and the free service!
